DuckDuckGoTests/domainMatching.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  domainMatching.swift
+//  DuckDuckGo
+//
+//  Copyright © 2021 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import XCTest
+@testable import TrackerRadarKit
+@testable import Core
+import Foundation
+
+struct RefTests: Decodable {
+    
+    struct Test: Decodable {
+        
+        let name: String
+        let siteURL: String
+        let requestURL: String
+        let requestType: String
+        let expectAction: String?
+        let exceptPlatforms: [String]?
+        
+    }
+    
+    struct DomainTests: Decodable {
+        
+        let name: String
+        let desc: String
+        let tests: [Test]
+        
+    }
+    
+    let domainTests: DomainTests
+}
+
+class DomainMatching: XCTestCase {
```

Comment:
This should be called `DomainMatchingTests` and the file name should match.

Reply:
Done!